src/components/TimerModal/CurrentTask.js

```diff
@@ -0,0 +1,23 @@
+export default function CurrentTask({task}) {
```

Comment:
This component can be replaced with the reusable task component which is in main now.